prod-sample/sample-modules/frank-lloyd-root/0.0.0/src/config.js

```diff
@@ -56,4 +56,5 @@ export default {
     ],
   },
   createSsrFetch: createFrankLikeFetch,
+  errorPageUrl: 'https://sample-cdn.frank/error.html',
```

Comment:
is this going to conflict with the other error page test ?

Reply:
It shouldn't

Reply:
Turns out it did, I resolved it.